app.rb

```diff
@@ -0,0 +1,122 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'classroom'
+
+class App
+  attr_accessor :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_people
+    @people.each { |person| puts "#{person.class}: Name: #{person.name}, Id: #{person.id}, Age: #{person.age}" }
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+
+    case person_type
+    when '1'
+      student = create_student
+      @people << student
+      puts 'Student created successfully'
+    when '2'
+      teacher = create_teacher
+      @people << teacher
+      puts 'Teacher created successfully'
+    else
+      puts 'Invalid option'
+    end
+  end
+
+  def create_student
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Age: '
+    age = gets.chomp.to_i
+
+    print 'Classroom: '
+    classroom_name = gets.chomp
+    classroom = Classroom.new(classroom_name)
+
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase == 'y'
+
+    Student.new(name, age, classroom, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly fix the error that shows when you want to create a student. I was able to create a teacher, but couldn't create a student.


![person](https://github.com/Alibaba2023/oop-school/assets/53124548/f5587cb4-2544-40c5-8ac9-f1d4a248007d)